src/main/java/core/basesyntax/services/strategy/impl/PurchaseHandlerOperation.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.services.strategy.impl;
+
+import core.basesyntax.db.DataStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.services.strategy.OperationHandler;
+
+public class PurchaseHandlerOperation implements OperationHandler {

```

Comment:
```suggestion
public class PurchaseOperationHandler implements OperationHandler {

```
check others as well